test/controllers/opensearch_controller_test.exs

```diff
@@ -0,0 +1,12 @@
+defmodule HexWeb.OpensearchControllerTest do
```

Comment:
We've recently changed project structure [1]; please change this to `Hexpm.Web.OpenSearchControllerTest` and update filename too.